src/cascadia/TerminalCore/TerminalApi.cpp

```diff
@@ -169,3 +169,49 @@ bool Terminal::SetColorTableEntry(const size_t tableIndex, const DWORD dwColor)
     _buffer->GetRenderTarget().TriggerRedrawAll();
     return true;
 }
+
+// Method Description:
+// - Sets the cursor style to the given style.
+// Arguments:
+// - cursorStyle: the style to be set for the cursor
+// Return Value:
+// - true iff we successfully set the cursor style
+bool Terminal::SetCursorStyle(const DispatchTypes::CursorStyle cursorStyle)
+{
+    CursorType finalCursorType = CursorType::Legacy;
+    bool shouldBlink = false;
+
+    switch (cursorStyle)
+    {
+    case DispatchTypes::CursorStyle::BlinkingBlock:
+    case DispatchTypes::CursorStyle::BlinkingBlockDefault:
+        finalCursorType = CursorType::FullBox;
+        shouldBlink = true;
+        break;
+    case DispatchTypes::CursorStyle::SteadyBlock:
+        finalCursorType = CursorType::FullBox;
+        shouldBlink = false;
+        break;
+    case DispatchTypes::CursorStyle::BlinkingUnderline:
+        finalCursorType = CursorType::Underscore;
+        shouldBlink = true;
+        break;
+    case DispatchTypes::CursorStyle::SteadyUnderline:
+        finalCursorType = CursorType::Underscore;
+        shouldBlink = false;
+        break;
+    case DispatchTypes::CursorStyle::BlinkingBar:
+        finalCursorType = CursorType::VerticalBar;
+        shouldBlink = true;
+        break;
+    case DispatchTypes::CursorStyle::SteadyBar:
+        finalCursorType = CursorType::VerticalBar;
+        shouldBlink = false;
+        break;
+    }

```

Comment:
The code should have a default case that either throws an exception for an unhandled case (if permissible), or if it really is intended that it does nothing if the type is unknkown, an explicit comment per the ISO C++ Guidelines:
[ES.79: Use default to handle common cases (only)](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es79-use-default-to-handle-common-cases-only)

Reply:
Added a `default` case.